Formula/freeimage.rb

```diff
@@ -25,6 +25,10 @@ class Freeimage < Formula
   end
 
   def install
+    if Hardware::CPU.arch == :arm64
```

Comment:
```suggestion
    if Hardware::CPU.arm?
```
Also, please leave a (brief) comment explaining the bug that this is meant to fix, along with a way to determine when this workaround can be removed. A link to an upstream issue could suffice.